lib/tilebag_class.rb

```diff
@@ -0,0 +1,38 @@
+# Create a `TileBag` class with a minimum of 5 specs. It should have the following class and instance methods:
+#
+require "./lib/scrabble_class.rb"
+require "./lib/player_class.rb"
+
+module Scrabble
+  class TileBag
+
+    attr_accessor :avaliable_tiles_array, :player_tiles, :total_player_tiles
+
+  # - `self.new` creates an instance with a collection of default tiles
+  def initialize
```

Comment:
Watch your indentation within this file, i think all of the methods have one less indent than they should